bedrock/base/templates/macros.html

```diff
@@ -584,6 +584,14 @@
           </ul>
         </li>
         <li>
+          <a href="{{ url('firefox') }}" class="primary-link" data-id="ios"><div>{{ _('<span>for</span> iOS') }}</div></a>
```

Comment:
Should probably wrap this whole thing in an <code>{% if waffle.switch('ios-active') %}</code>